Source/Extensions/Blazorise.DataGrid/Models/DataGridColumnInfo.cs

```diff
@@ -57,4 +59,9 @@ public DataGridColumnInfo( string field, object searchValue, SortDirection sortD
     /// Gets the column type.
     /// </summary>
     public DataGridColumnType ColumnType { get; }
+
+    /// <summary>
+    /// Gets the column filter method.
+    /// </summary>
+    public DataGridFilterMethod? FilterMethod { get; }
```

Comment:
Why the new field is nullable?

Reply:
If `FilterMethod` is null. It Means that the filter has not been set for this column.

Parity with how filtering is done when it's static data:
![image](https://github.com/Megabit/Blazorise/assets/22283161/6529e6bb-4082-43cb-938a-347cbb78d8bf)
![image](https://github.com/Megabit/Blazorise/assets/22283161/9db0974a-f7fb-433b-b7b3-f61efde72ae7)

And to what's defined in the column:
![image](https://github.com/Megabit/Blazorise/assets/22283161/418a5660-346c-4c0c-b4a5-13dbe0af4dde)

Reply:
Cotcha!